src/main/resources/sample/helloworld-settings.yml

```diff
@@ -1,10 +1,7 @@
 # This name is only used as a response string during init and is purely cosmetic
 extensionName: hello-world
-# The listening port is bound to the interface with this IP. Set to 0.0.0.0 to listen on all interfaces
-hostAddress: 0.0.0.0
```

Comment:
Im curious as to how this modification passed the checks. Won't this modification break this test, since the expected extension hostAddress setting for this test is 0.0.0.0: https://github.com/opensearch-project/opensearch-sdk-java/blob/579c3020cc2775df1cf014b67ad6ef8f61b27766/src/test/java/org/opensearch/sdk/sample/helloworld/TestHelloWorldExtension.java#L105

Reply:
That must have happened as part of a merge - I didn't change the address at all

Reply:
let's also keep the comments on this file

Reply:
Fixed